lib/noether.ex

```diff
@@ -3,41 +3,47 @@ defmodule Noether do
   Noether aims to ease common data manipulation tasks by introducing simple algebraic functions and other utilities.
   Functions and names are inspired (sometimes taken as-is) from Haskell.
 
-  The `Maybe` module introduces operations on nullable values.
-  The `Either` module introduces operations on `{:ok, _} | {:error, _}` values.
-  The `List` module introduces operations on lists.
+  The `Noether.Maybe` module introduces operations on nullable values.    
```

Comment:
The tab at the end

Reply:
I wanted to force a newline, is there a better way?

Reply:
New line for the MD? Ok, fine